Examples/Sources/MacroExamples/Implementation/Expression/AddBlocker.swift

```diff
@@ -72,17 +72,12 @@ public struct AddBlocker: ExpressionMacro {
             )
           )
 
-          return ExprSyntax(
-            node.with(
-              \.operator,
-              ExprSyntax(
-                binOp.with(
-                  \.operator,
-                  binOp.operator.with(\.tokenKind, .binaryOperator("-"))
-                )
-              )
-            )
-          )
+          binOp.operator.tokenKind = .binaryOperator("-")
+
+          var node = node
+          node.operator = ExprSyntax(binOp)
+
+          return ExprSyntax(node)
```

Comment:
I think in this case where we really are creating a copy with only one property modified, the `with` syntax reads nicer.

```suggestion
          return ExprSyntax(node.with(\.operator,  ExprSyntax(binOp)))
```

Reply:
Totally agree with you! Initially, I wasn't sure if you were suggesting changing all occurrences of `with` or just specific ones. But this one is definitely a keeper! :D